response_operations_ui/static/scss/components/search.scss

```diff
@@ -1,9 +0,0 @@
-.ru-search-box {
```

Comment:
This is needed

Reply:
ru search box is being changed, I discussed these changes with @nicprice on Thursday